cocos/3d/lights/directional-light-component.ts

```diff
@@ -460,6 +461,15 @@ export class DirectionalLight extends Light {
     constructor () {
         super();
         this._lightType = scene.DirectionalLight;
+
+        const highQualityMode = settings.querySettings('rendering', 'highQualityMode');
+
+        if (highQualityMode) {
+            this._shadowPcf = PCFType.SOFT_2X;
+            this._shadowDistance = 200;
```

Comment:
shadowDistance should be 50, the higher the value, the lower the shadow quality
this._csmEnable = true;